ui/dashboards/src/components/PanelGroupDialog/PanelGroupDialog.tsx

```diff
@@ -61,7 +61,7 @@ export function PanelGroupDialog() {
           >
             <CloseIcon />
           </IconButton>
-          <DialogContent sx={{ width: '500px' }}>
+          <DialogContent dividers sx={{ width: '500px' }}>
```

Comment:
Can you also update the cancel button to have `outlined` variant? 

similar to what we have in Discard Changes Confirmation Dialog 

![image](https://user-images.githubusercontent.com/9817826/210822498-d6011b42-d1f9-4552-8920-3310cba4a647.png)

Reply:
Absofruitly